apiv2/pkg/clients/configrations/configration.go

```diff
@@ -0,0 +1,65 @@
+package configrations
```

Comment:
```suggestion
package configurations
```

This typo is not only in the package name, but also in the filenames.